src/PostalCodeLocator.php

```diff
@@ -42,6 +42,12 @@ public function __construct (PostalCodeAdapterInterface $adapter)
     public function loadNearPostalCodes ($postalCode, $radius, $limit = null)
     {
         $postalCodeDetails = $this->loadPostalCodeDetails($postalCode);
+
+        if (null === $postalCodeDetails)
```

Comment:
If you check for `empty` here, you can omit the addition in `loadPostCodeDetails`.